src/storage/FileResourceMapper.ts

```diff
@@ -0,0 +1,48 @@
+import { types } from 'mime-types';
+import { ResourceIdentifier } from '../ldp/representation/ResourceIdentifier';
+import { NotFoundHttpError } from '../util/errors/NotFoundHttpError';
+import { FileResourceStore } from './FileResourceStore';
+import { ResourceMapper } from './ResourceMapper';
+
+export class FileResourceMapper implements ResourceMapper {
+  private readonly fileStore: FileResourceStore;
+  private readonly types: Record<string, any>;
+
+  public constructor(fileStore: FileResourceStore, overrideTypes = { acl: 'text/turtle', metadata: 'text/turtle' }) {
+    this.fileStore = fileStore;
+    this.types = { ...types, ...overrideTypes };
+  }
+
+  /**
+   * Strips the baseRequestURI from the identifier and checks if the stripped base URI matches the store's one.
+   * @param identifier - Incoming identifier.
+   *
+   * @throws {@link NotFoundHttpError}
+   * If the identifier does not match the baseRequestURI path of the store.
+   */
+  public mapUrlToFilePath(identifier: ResourceIdentifier): string {
```

Comment:
"map" isn't completely correct since the result is not the file path (since the base file path doesn't get added).

Reply:
So let's add the base path 😄